arm-resources/resources/2016-02-01/swagger/resources.json

```diff
@@ -1877,6 +1877,22 @@
         "properties": {
           "type": "object",
           "description": "Gets or sets the resource properties."
```

Comment:
The prefix text `Gets or sets` _really_ shouldn't be there. The code generator will generate that for languages that support properties.